### PR TITLE
add Action classifiers

### PR DESCRIPTION
**Pull request by jwgwalton**
Do I bump the snapshot version on release?

### CODE REVIEW

**Comment by benjefferies** [comment]
@jwgwalton none it will all be handled for you on the merge to master 👍

**Comment by jwgwalton** [comment]
@rjweeks This is the domain model from the messages

File: src/main/resources/actionsvc/xsd/actionInstruction.xsd [hunk 1]
```diff
@@ -65,6 +65,26 @@
...
 	</xs:complexType>
 
+	<!-- a complex type for classifiers-->
+	<xs:complexType name="ActionClassifiers">
+		<xs:sequence>
+			<xs:element name="legalBasis" type="xs:string" minOccurs="0"
+						maxOccurs="1" />
+			<xs:element name="region" type="xs:string" minOccurs="0"
+						maxOccurs="1" />
+			<xs:element name="respondentAccountStatus" type="xs:string" minOccurs="0"
```
Comment (rjweeks) [inline src/main/resources/actionsvc/xsd/actionInstruction.xsd]: I think we should separate Classifiers from State. Classifiers are currently just legalBasis and region.

File: src/main/resources/actionsvc/xsd/actionInstruction.xsd [hunk 2]
```diff
@@ -65,6 +65,26 @@
 		</xs:sequence>
 	</xs:complexType>
 
+	<!-- a complex type for classifiers-->
+	<xs:complexType name="ActionClassifiers">
```
Comment (rjweeks) [inline src/main/resources/actionsvc/xsd/actionInstruction.xsd]: I think we can just call it Classifiers

File: src/main/resources/actionsvc/xsd/actionInstruction.xsd [hunk 3]
```diff
@@ -65,23 +65,36 @@
...
+	<xs:complexType name="Classifiers">
+		<xs:sequence>
+			<xs:element name="legalBasis" type="xs:string" minOccurs="0" maxOccurs="1" />
+			<xs:element name="region" type="xs:string" minOccurs="0" maxOccurs="1" />
+		</xs:sequence>
+	</xs:complexType>
+
+	<xs:complexType name="Statuses">
+		<xs:sequence>
+		<xs:element name="respondentAccountStatus" type="xs:string" minOccurs="0" maxOccurs="1" />
```
Comment (rjweeks) [inline src/main/resources/actionsvc/xsd/actionInstruction.xsd]: RespondentStatus

File: src/main/resources/actionsvc/xsd/actionInstruction.xsd [hunk 4]
```diff
@@ -65,23 +65,36 @@
...
+		<xs:element name="enrolmentStatus" type="xs:string" minOccurs="0" maxOccurs="1"/>
+		<xs:element name="responseState" type="xs:string" minOccurs="0" maxOccurs="1" />
```
Comment (rjweeks) [inline src/main/resources/actionsvc/xsd/actionInstruction.xsd]: CaseGroupState

File: src/main/resources/actionsvc/xsd/actionInstruction.xsd [hunk 5]
```diff
@@ -65,23 +65,36 @@
 		</xs:sequence>
 	</xs:complexType>
 
+	<!-- a complex type for classifiers-->
+	<xs:complexType name="Classifiers">
```
Comment (rjweeks) [inline src/main/resources/actionsvc/xsd/actionInstruction.xsd]: CommsTypeClassifiers

**Pull request merged**